docs/performance/inserts/bulk.rst

```diff
@@ -0,0 +1,419 @@
+.. highlight:: psql
+
+.. _bulk-inserts:
+
+============
+Bulk inserts
+============
+
+If you have a lot of pre-existing data that you need to import in bulk, follow
+this how-to guide if you want to improve performance.
```

Comment:
```suggestion
this how-to guide to get optimal performance on this task.
```

Reply:
I think I like my phrasing better, for conciseness